README.md

```diff
@@ -1 +1,41 @@
-# horizen-2-doc
\ No newline at end of file
+# Website
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
```

Comment:
I know this is wrong in the zkVerify-docs, but let's update the instructions here to use `npm` if that is what will be used in the CI

Reply:
Or update .travis.yml to use yarn. I don't have a specific preference.